ui/app/AppLayouts/Chat/controls/community/InDropdown.qml

```diff
@@ -154,27 +151,26 @@ StatusDropdown {
 
             visible: root.allowChoosingEntireCommunity
         }
-
         StatusScrollView {
             id: scrollView
-
             Layout.fillWidth: true
-            Layout.fillHeight: true
-            Layout.bottomMargin: 9
+            Layout.minimumHeight: (((topRepeater.count > d.maxHeightCountNo) ? d.maxHeightCountNo : topRepeater.count)*d.itemStandardHeight)
```

Comment:
It would be simpler and more readable imo:
```
Layout.minimumHeight: Math.min(d.maxHeightCountNo, topRepeater.count) * d.itemStandardHeight
```

Reply:
definitely!